gdal/frmts/netcdf/netcdfdataset.cpp

```diff
@@ -3392,6 +3403,11 @@ void netCDFDataset::SetProjectionFromVar( int nGroupId, int nVarId,
         // variables without same name than dimension using the same resolving
         // logic. This should handle for example NASA Ocean Color L2 products.
 
+        const bool bIgnoreXYAxisNameChecks =
+            CPLTestBool(CPLGetConfigOption("GDAL_NETCDF_IGNORE_XY_AXIS_NAME_CHECKS", "NO")) ||
```

Comment:
`GDAL_NETCDF_IGNORE_XY_AXIS_NAME_CHECKS` will be very helpful for backwards compatibility. Thanks :+1:. Hopefully it won't be needed for datasets moving forward :crossed_fingers: